### ISSUE TITLE
Implement the /api/status service endpoint.

### ISSUE THREAD

**Issue by kordless** [WisdomAPI/site#1]
Implement the status resource endpoint for //wisdom.sh/api/status. A GET request plus response:

```
curl -X GET https://api.wisdom.sh/api/status

{  
  "type": "GET",
  "status": "200",
  "view": "https://wisdom.sh/api/status",
  "timestamp": "2017-09-01 12:00:00",
  "qps": "2.0"
}
```

**Comment by kordless** [comment]
Added in temporary path handler. 498e766552569e15098d4221bb3c063bd02e4b50

**Comment by kordless** [comment]
https://github.com/WisdomAPI/deployment/issues/2 and https://github.com/WisdomAPI/deployment/issues/1 need to be completed next.

**Comment by kordless** [comment]
Implemented with f2ff44a..6a0d30f
